requirements.txt

```diff
@@ -1,9 +1,4 @@
 six
```

Comment:
Please see comment on PR.

Reply:
hi Robert, 
1.remove the six and use import urllib to import urljoin and so on.
2. use comma can fix the passenv error, so try to use it instead of removing.